draft-ietf-tls-extended-key-update.md

```diff
@@ -512,13 +512,101 @@ Auth | {CertificateVerify}
 
 #  DTLS 1.3 Considerations
 
+Unlike TLS 1.3, DTLS 1.3 implementations must take into account that handshake
+messages are not transitted over a reliable transport protocol. As with other
+handshake messages with no built-in response, NewKeyUpdate messages MUST be
+acknowledged. Unlike the NewKeyUpdate message, the ExtendedKeyUpdateRequest
+is acknowledged by the ExtendedKeyUpdateResponse.
+
+The exchange has the following steps:
+
+1. Initiator sends a ExtendedKeyUpdateRequest message, which contains
+a key share. While an extended key update is in progress, the initiator
+MUST NOT initiate further key updates.
+
+2. On receipt of the ExtendedKeyUpdateRequest message, the responder
+sends the ExtendedKeyUpdateResponse message. If the responder accepts the
+request, it sets the status to `accepted` and includes its own key share.
+If the responder declines the request, it sets the status accordingly and
+does not include the key share. While an extended key update is in progress,
+the responder MUST NOT initiate further key updates.
+
+3. On receipt of the ExtendedKeyUpdateResponse message with `accepted` status,
+the initiator is able to derive a secret key based on the exchanged key shares.
+The NewKeyUpdate message is intentionally an empty structure that triggers
+the transition to new keying material.
+
+4. The initiator transmits the NewKeyUpdate message.
+
+5. Upon receiving a NewKeyUpdate message, the responder MUST update its receive
+keys. It then acknowledges the received message by sending its own NewKeyUpdate
+message. This response both acknowledges the incoming key update and initiates
+a new key update in a single message, as the acknowledgment is piggybacked.
+
+6. After the initiator receives the ACK from the responder, the initiator
+MUST update its send key. With the receipt of the NewKeyUpdate message the
```

Comment:
Are you exchanging ACK and NewKeyUpdate? if yes, why do you describe the sequence of receiving different that the sequence of sending?